### ISSUE TITLE
Console Error

### ISSUE THREAD

**Issue by Anoniempje1234** [SkriptLang/Skript#2471]
### Errors / Screenshots
```
>.... [20:38:34 ERROR]: #!#! Stack trace:
>.... [20:38:34 ERROR]: #!#! java.lang.NullPointerException
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.util.slot.InventorySlot.getItem(InventorySlot.java:65)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.classes.data.DefaultComparators$4.compare(DefaultComparators.java:154)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.classes.data.DefaultComparators$4.compare(DefaultComparators.java:1)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.conditions.CondCompare.lambda$1(CondCompare.java:257)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.util.SimpleExpression.check(SimpleExpression.java:173)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.util.SimpleExpression.check(SimpleExpression.java:161)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.util.SimpleExpression.check(SimpleExpression.java:156)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.conditions.CondCompare.lambda$0(CondCompare.java:254)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.util.SimpleExpression.check(SimpleExpression.java:173)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.util.SimpleExpression.check(SimpleExpression.java:161)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.conditions.CondCompare.check(CondCompare.java:253)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.Condition.run(Condition.java:56)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.Conditional.walk(Conditional.java:51)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.lang.Trigger.execute(Trigger.java:57)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:156)
>.... [20:38:34 ERROR]: #!#! at ch.njol.skript.SkriptEventHandler$1.execute(SkriptEventHandler.java:114)
>.... [20:38:34 ERROR]: #!#! at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
>.... [20:38:34 ERROR]: #!#! at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
>.... [20:38:34 ERROR]: #!#! at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:2310)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:33)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.PacketPlayInWindowClick.a(SourceFile:10)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:23)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.MinecraftServer.aX(MinecraftServer.java:1013)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:1006)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:990)
>.... [20:38:34 ERROR]: #!#! at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:923)
>.... [20:38:34 ERROR]: #!#! at java.lang.Thread.run(Thread.java:748)
>.... [20:38:34 ERROR]: #!#!
>.... [20:38:34 ERROR]: #!#! Version Information:
>.... [20:38:34 ERROR]: #!#! Skript: 2.4-beta7 (latest)
>.... [20:38:34 ERROR]: #!#! Flavor: skriptlang-github
>.... [20:38:34 ERROR]: #!#! Date: 2019-09-05
>.... [20:38:34 ERROR]: #!#! Bukkit: 1.14.4-R0.1-SNAPSHOT
>.... [20:38:34 ERROR]: #!#! Minecraft: 1.14.4
>.... [20:38:34 ERROR]: #!#! Java: 1.8.0_212 (OpenJDK 64-Bit Server VM 25.212-b01)
>.... [20:38:34 ERROR]: #!#! OS: Linux amd64 4.15.0-46-generic
>.... [20:38:34 ERROR]: #!#!
>.... [20:38:34 ERROR]: #!#! Server platform: Paper
>.... [20:38:34 ERROR]: #!#!
>.... [20:38:34 ERROR]: #!#! Current node: null
>.... [20:38:34 ERROR]: #!#! Current item: event-slot is equal to [[itemtype:sugar cane]] named "Page 1" with lore "Go to page 1 of %{at.target.%the name of the player%}(as java.lang.Object)%'s AdminTool" (comparator: ch.njol.skript.classes.data.DefaultComparators$4@6cc201d)
>.... [20:38:34 ERROR]: #!#! Current trigger: inventory click (dispense/spawn/drop/craft/pickup/consume/break/despawn/merge) (admin.sk, line 435)
>.... [20:38:34 ERROR]: #!#!
>.... [20:38:34 ERROR]: #!#! Thread: Server thread
>.... [20:38:34 ERROR]: #!#!
>.... [20:38:34 ERROR]: #!#! Language: english
>.... [20:38:34 ERROR]: #!#! Link parse mode: DISABLED
>.... [20:38:34 ERROR]: #!#!
>.... [20:38:34 ERROR]: #!#! End of Error.
```

### Server Information
* **Server version/platform:**       Paper 1.14.4
* **Skript version:**                2.4-beta7

### Additional Context
I don't think something goes wrong when I use the skript... It's just annoying I have it all the time in the console. :(

**Comment by Wealthyturtle** [comment]
Should be fixed in #2423, in the next update

**Comment by Anoniempje1234** [comment]
Okay, thanks!
